examples/repl.rs

```diff
@@ -129,10 +192,14 @@ mod example {
                         println!("Exit");
                         return;
                     }
-                    match run_command(&mut context, &command) {
+                    match run_command(&mut context, &color, &command) {
                         Ok(()) => {}
                         Err(err) => {
-                            println!("\x1b[91mError: {}\x1b[0m", err);
+                            if color.get() {
+                                println!("\x1b[91mError: {}\x1b[0m", err);
```

Comment:
Maybe use a dev-dependency like [`ansi_term`](https://crates.io/crates/ansi_term) to specify colours?

Reply:
`ansi_term` doesn't have a method for the bright variants of colors: https://github.com/ogham/rust-ansi-term/issues/66